pkg/node/manager/manager.go

```diff
@@ -132,12 +137,20 @@ func (m *manager) AddNode(nodeName string) error {
 	var newNode node.Node
 	var nodeFound bool
 
-	newNode, nodeFound = m.dataStore[k8sNode.Name]
+	_, nodeFound = m.dataStore[k8sNode.Name]
 	if nodeFound {
 		log.Info("node is already processed, not processing add event again")
 		return nil
 	}
 
+	if err = m.CreateCNINodeIfNotExisting(k8sNode); err != nil {
+		m.Log.Error(err, "Failed to create CNINode for k8sNode", "NodeName", k8sNode.Name)
+		return err
+	} else {
+		// use an event to notify users that a CNINode has been created for this node
+		utils.SendNodeEventWithNodeObject(m.wrapper.K8sAPI, k8sNode, utils.CNINodeCreatedReason, "A new CNINode was created for this node", v1.EventTypeNormal, m.Log)
```

Comment:
Do we need to create a kube event for this? Seems like it may expose unnecessary information to customers, and they can watch the CNI Node object if they need.

Reply:
Make sense, we don't have to. I will remove this event.